src/core/directives/google-map.ts

```diff
@@ -79,6 +79,11 @@ export class SebmGoogleMap implements OnChanges, OnInit {
   zoom: number = 8;
 
   /**
+   * Enables/disables if map is draggable.
+   */
+  draggable: boolean = true;
```

Comment:
See this example in the marker: https://github.com/SebastianM/angular2-google-maps/blob/master/src/core/directives/google-map-marker.ts#L40